src/engine/renderer/tr_local.h

```diff
@@ -1049,9 +1049,7 @@ enum class dynamicLightRenderer_t { LEGACY, TILED };
 	struct expression_t
 	{
 		expOperation_t ops[ MAX_EXPRESSION_OPS ];
-		uint8_t        numOps;
-
-		bool       active; // no parsing problems
+		uint8_t numOps;
```

Comment:
Should we use `int`?

Reply:
Yes

Reply:
I now use `size_t` for `numOps` and related things everywhere.